runners/google-cloud-dataflow-java/examples/build.gradle

```diff
@@ -42,57 +42,60 @@ def dockerImageName = project(':beam-runners-google-cloud-dataflow-java').ext.do
 // If -PuseExecutableStage is set, the use_executable_stage_bundle_execution wil be enabled.
 def fnapiExperiments = project.hasProperty('useExecutableStage') ? 'beam_fn_api,use_executable_stage_bundle_execution' : "beam_fn_api"
 
+// we require dataflowWorker, optional workerHarnessContainerImage and optional additionalOptions
```

Comment:
Isn't this comment redundant? The line below shows exactly what the comment says but (and that's the risk) it can change in the future. 

Please delete the comment

Reply:
Ok, done